### ISSUE TITLE
Create basic topic/message frontend

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#23]
## User story
1. As a user
2. I need a simple and userfriendly interface
3. So that I can see all topics and their corresponding messages and send new messages to a topic.

## Acceptance criteria
* [ ] Create dummy data for topics/messages (match possible backend data format)
* [ ] Display all existing topics (each can be selected)
* [ ] When topic is selected, list all messages
* [ ] Create form where user can send a new message to that topic (add to dummy data)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
